### PR TITLE
Merge several fixes and cleanup 

### PR DESCRIPTION
**Pull request by amgadhanafy**
### Describe the pull request:

- [x] Bug fix
- [x] Code cleanup

### CODE REVIEW

**Review by dtitov — APPROVED** [review]

File: ega-data-api-filedatabase/src/main/java/eu/elixir/ega/ebi/downloader/service/internal/LocalEGAFileServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+
```
Comment (blankdots) [inline ega-data-api-filedatabase/src/main/java/eu/elixir/ega/ebi/downloader/service/internal/LocalEGAFileServiceImpl.java]: this should not be here, it was removed in: https://github.com/neicnordic/ega-data-api/pull/3

File: extras/travis_push_docker_hub.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (blankdots) [inline extras/travis_push_docker_hub.sh]: was removed in: https://github.com/neicnordic/ega-data-api/pull/2

**Review by blankdots — CHANGES_REQUESTED** [review]
two files was previously removed, we should keep it like that.
We might want to review the log messages , as they are not really that informative to be relevant in any production environment, e.g.:
>  log.info("NO!!");

Will start an issue on that.

**Comment by blankdots** [comment]
Follow up issue created: https://github.com/neicnordic/ega-data-api/issues/11

**Review by blankdots — APPROVED** [review]

**Pull request merged**